src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,673 @@ description: Learn how to use images in Astro.
 i18nReady: true
 ---
 import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
 
 Astro provides several ways for you to use images on your site, whether they are stored locally inside your project, linked to remotely, or stored in a CMS or CDN!
```

Comment:
I guess this was already here, but "linked to remotely" sounds off? I would even say that remote images are the same thing as using with a  CMS / CDN? Is there any technical difference that I am missing?

```suggestion
Astro provides several ways for you to use images on your site, whether they are stored locally inside your project, or linked remotely through a CMS or CDN!
```

Reply:
Yeah, I felt that too this time around. BUT, there is a random, one-off inline `src="url"` vs using a CMS/CDN. They aren't the same thing so I don't want to make it sound like you can't just use a random URL. I'll think about it while I apply the Wrath of the Commas.